src/android/ContentFilesystem.java

```diff
@@ -193,7 +193,7 @@ protected Long lastModifiedDateForCursor(Cursor cursor) {
             	return Long.parseLong(dateStr);
             }
         }
-        return null;
+        return 0;
```

Comment:
Should this return 0? Perhaps the null result can be converted to [JSONObject.NULL](http://developer.android.com/reference/org/json/JSONObject.html#NULL) in `lastModifiedDateForCursor()` to avoid special return values.